### PR TITLE
Allow stable versions of async-aws

### PR DESCRIPTION
**Pull request by Nyholm**

### CODE REVIEW

**Comment by mnapoli** [comment]
👍  to merge but there are errors from phpstan

File: src/Service/Sns/SnsTransport.php [hunk 1]
```diff
@@ -33,12 +33,9 @@ public function send(Envelope $envelope): Envelope
...
         $arguments = [
-            'MessageAttributes' => new MessageAttributeValue([
-                'Headers' => [
-                    'DataType' => 'String',
-                    'StringValue' => json_encode($headers, JSON_THROW_ON_ERROR),
-                ],
-            ]),
+            'MessageAttributes' => [
+                'Headers' => new MessageAttributeValue(['DataType' => 'String', 'StringValue' => json_encode($headers, JSON_THROW_ON_ERROR)]),
+            ],
```
Comment (Nyholm) [inline src/Service/Sns/SnsTransport.php:38]: This is a bug fix

**Comment by mnapoli** [comment]
Thanks!

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**